openc3-cosmos-cmd-tlm-api/app/controllers/script_autocomplete_controller.rb

```diff
@@ -26,12 +26,9 @@ class ScriptAutocompleteController < ApplicationController
   CMD_KEYWORDS = %w(cmd cmd_no_range_check cmd_no_hazardous_check cmd_no_checks
                     cmd_raw cmd_raw_no_range_check cmd_raw_no_hazardous_check cmd_raw_no_checks)
 
-  TLM_KEYWORDS = %w(set_tlm set_tlm_raw override_tlm override_tlm_raw normalize_tlm
-                    tlm tlm_raw tlm_formatted tlm_with_units
+  TLM_KEYWORDS = %w(set_tlm override_tlm normalize_tlm tlm
                     limits_enabled? enable_limits disable_limits
-                    check check_raw check_tolerance check_tolerance_raw
-                    wait wait_raw wait_tolerance wait_tolerance_raw wait_check wait_check_raw
-                    wait_check_tolerance wait_check_tolerance_raw)
+                    check wait wait_tolerance wait_check wait_check_tolerance)
```

Comment:
I removed deprecated keywords and will also update the website with deprecation notices

Reply:
check_tolerance missing?